NomaiGrandPrix/SpawnPoints.tsv

```diff
@@ -45,109 +59,95 @@ Spawn_DreamZone_3_SecretGallery	DreamZone 3 Secret Gallery	DreamZone	TRUE	FALSE
+Spawn_QuantumMoonLocator	Quantum Moon Locator	EmberTwin	FALSE	TRUE	TRUE	FALSE	
+Spawn_SandslideCave	Sandslide Cave	EmberTwin	FALSE	TRUE	TRUE	FALSE	
+Spawn_City	Sunless City	EmberTwin	FALSE	TRUE	TRUE	FALSE	
+Spawn_PodPath	Sunless City Trail	EmberTwin	FALSE	TRUE	TRUE	FALSE	
+Spawn_WarpReceiver	Warp Receiver	EmberTwin	FALSE	TRUE	TRUE	FALSE	
+Spawn_BrambleIsland	Bramble Island	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+Spawn_ConstructionYard	Construction Yard	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+Spawn_Core	Core	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+Spawn_GabbroIsland	Gabbro's Island	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+ShipSpawn_GD	Giant's Deep (Orbit)	GiantsDeep	FALSE	FALSE	FALSE	FALSE	
+Spawn_DeepSpace	Giant's Deep (Orbit)	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+ShipSpawn_JellyDomain	Jelly Domain	GiantsDeep	FALSE	FALSE	FALSE	FALSE	Duplicate
+Spawn_JellyDomain	Jelly Domain	GiantsDeep	FALSE	TRUE	TRUE	FALSE	Northern Half of GD
+Spawn_OPC	Probe Cannon	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+Spawn_Module_Broken	Probe Cannon Broken Module	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+Spawn_Module_Intact	Probe Cannon Intact Module	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+Spawn_Module_Sunken	Probe Cannon Sunken Module	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+Spawn_StatueIsland_Beach	Statue Island Beach	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+Spawn_StatueIsland_Inside	Statue Island Inside	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+Spawn_StatueIsland_Top	Statue Island Top	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+Spawn_QuantumIsland_Bottom	Tower of Quantum Trials	GiantsDeep	FALSE	TRUE	TRUE	FALSE	Outside by elevator
+Spawn_QuantumIsland_PuzzleRoom4	Tower of Quantum Trials Bottom	GiantsDeep	FALSE	TRUE	TRUE	FALSE	End of last solved puzzle
+Spawn_QuantumIsland_Top	Tower of Quantum Trials Top	GiantsDeep	FALSE	TRUE	TRUE	FALSE	Before entering first puzzle
+Spawn_CO	Interloper	Interloper	FALSE	TRUE	TRUE	FALSE	
+ShipSpawn_CO	Interloper (Orbit)	Interloper	FALSE	FALSE	FALSE	FALSE	
+Spawn_CO_Core	Interloper Core	Interloper	FALSE	TRUE	TRUE	FALSE	
+Spawn_CO_Interior	Interloper Interior	Interloper	FALSE	TRUE	TRUE	FALSE	
+Spawn_IP_Zone_2	Cinder Isles	Stranger	FALSE	FALSE	TRUE	FALSE	
+Spawn_IP_Zone_2_CoveBottom	Cinder Isles Cove Bottom	Stranger	FALSE	FALSE	TRUE	FALSE	
+Spawn_IP_Zone_2_CoveTop	Cinder Isles Cove Top	Stranger	FALSE	FALSE	TRUE	FALSE	
+Spawn_IP_Zone_2_LighthouseTop	Cinder Isles Lighthouse Top	Stranger	FALSE	FALSE	TRUE	FALSE	
+Spawn_IP_DockingBay_Dark	Docking Bay Dark	Stranger	FALSE	FALSE	TRUE	FALSE	
+Spawn_IP_DockingBay_Light	Docking Bay Light	Stranger	FALSE	FALSE	TRUE	FALSE	
+Spawn_IP_Zone_3_Bridge	Hidden Gorge Bridge	Stranger	FALSE	FALSE	TRUE	FALSE	
+Spawn_IP_Zone_3_BrokenBridge	Hidden Gorge Broken Bridge	Stranger	FALSE	FALSE	TRUE	FALSE	
+Spawn_IP_Zone_3_CodeDoor	Hidden Gorge Code Door	Stranger	FALSE	FALSE	TRUE	FALSE	
+Spawn_IP_Zone_3_DreamFire	Hidden Gorge Dream Fire	Stranger	FALSE	FALSE	TRUE	FALSE	
+Spawn_IP_Zone_3_DreamLabOverlook	Hidden Gorge DreamLab Overlook	Stranger	FALSE	FALSE	TRUE	FALSE	
```

Comment:
maybe "Dream Lab"?

Reply:
I didnt really mess with the stranger ones much, I will check but i think this has a Canon name "Artifact workshop" or something

Reply:
Just said Laboratory